packages/insured-bridge-relayer/src/index.ts

```diff
@@ -27,28 +28,44 @@ export async function run(logger: winston.Logger, web3: Web3): Promise<void> {
     logger[config.pollingDelay === 0 ? "debug" : "info"]({
       at: "InsuredBridgeRelayer#index",
       message: "Relayer started 🌉",
-      bridgePoolFactoryAddress: config.bridgeAdmin,
+      config,
     });
 
-    const [accounts, networkId] = await Promise.all([web3.eth.getAccounts(), web3.eth.net.getId()]);
-    console.log(`connected to ${accounts[0]} on ${networkId}`); // just show web3 connection works.
+    const [accounts] = await Promise.all([l1Web3.eth.getAccounts()]);
 
     const gasEstimator = new GasEstimator(logger);
 
     // Create L1/L2 clients to pull data to inform the relayer.
     // todo: add in start and ending block numbers (if need be).
-    const l1Client = new InsuredBridgeL1Client(logger, web3, config.bridgeAdmin, config.rateModels);
+    const l1Client = new InsuredBridgeL1Client(logger, l1Web3, config.bridgeAdmin, config.rateModels);
+
+    // TODO: Add a method to fetch all registered chainIDs from bridge admin to let the bot default to all chains when
```

Comment:
note for now we only use the chainIds provided by the user. in the future this will auto-detect all loaded chain-ids on L1 and runn accordingly.